File: app/src/main/java/com.caique.everis.testeandroid/model/LoginResponse.java
package com.caique.everis.testeandroid.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class LoginResponse implements Serializable {

    @SerializedName("userAccount")
    private ClientData clientData;

    public ClientData getClientData() {
        return clientData;
    }

    public void setClientData(ClientData clientData) {
        this.clientData = clientData;
    }
}

File: teste/app/src/main/java/com/example/ilealnod/SantanderProjectKotlin/Todos/Dependencias/LoginResponse.kt
package com.example.ilealnod.SantanderProjectKotlin.Todos.Dependencias

import com.google.gson.annotations.SerializedName

// Classe que pega os Dados do cliente / resposta da APi

class LoginResponse {

    @SerializedName("userAccount")
    var clientData: ClientData? = null
}
